src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -609,6 +625,22 @@ public override ushort UInt16Value
                 }
             }
 
+            public override int Int32Value
```

Comment:
>public override int Int32Value [](start = 12, length = 30)

The same question is for UInt64Value property, the following code prints three different numbers, which one is the expected result?
```
                short x = -1;
                System.Console.WriteLine((ulong)x);
                System.Console.WriteLine((ulong)(uint)x);
                System.Console.WriteLine((ulong)(ushort)x);
```